libretro-rs-sys/build.rs

```diff
@@ -13,6 +13,7 @@ fn main() {
     .allowlist_type("^retro_.+$")
     .allowlist_var("^RETRO_.+$")
     .default_enum_style(bindgen::EnumVariation::Rust { non_exhaustive: false })
+    .derive_default(true)
```

Comment:
This adds the Default trait to the C structs which is very useful for the environment `raw` methods.